--- src/inrimage/inr_zcat.c.orig	2015-02-25 08:20:41.000000000 +0100
+++ src/inrimage/inr_zcat.c	2015-02-25 08:19:45.000000000 +0100
@@ -3,6 +3,7 @@
 #include <inrimage/error.h>
 #include <stdio.h>
 #include <unistd.h>
+#include <crt_externs.h>
 #if 0 // bereziat 10/2006
 #ifdef sun5
 #    include <string.h>
@@ -17,7 +18,6 @@
 extern int  debug_;
 extern char *inr_mymalloc(), *inr_myrealloc(), *inr_mycalloc();
 static void inr_rdexts();
-extern char **environ;
 extern char *get_inr_extfile();
 
 extern char inr_force_ext[];  /* pour forcer l'ext des images d'netree */
@@ -308,6 +308,7 @@
 #if 0
 		execve(p->cmde,tabarg,(char **)0);
 #else
+                char** environ = *_NSGetEnviron();
 		execve(p->cmde,tabarg,environ);
 #endif
 		imerror(9,"image_ : execve err %d\n",errno);
