--- ./src/contlib7/iouvct.c.orig	2015-03-05 09:36:25.000000000 +0100
+++ ./src/contlib7/iouvct.c	2015-03-05 09:36:31.000000000 +0100
@@ -202,6 +202,7 @@
  * @param nfc descripteur de fichier de contours
  * @sa c_iouvct()
  */
+void
 c_fermct(nfc)
 struct fc_cont *nfc;
 {
--- ./src/contlib7/wrcttb.c.orig	2015-03-05 09:38:24.000000000 +0100
+++ ./src/contlib7/wrcttb.c	2015-03-05 09:39:01.000000000 +0100
@@ -38,6 +38,7 @@
  * @param icom commentaire à écrire (tampon d'au plus 128 caracètes).
  */
 
+void
 c_wctcom(nfc,icom)
 struct fc_cont *nfc;
 char *icom;
